FirebaseAppDistribution/Sources/FIRAppDistributionUIService.m

```diff
@@ -244,16 +234,16 @@ - (void)resetUIState {
 
   self.registrationFlowCompletion = nil;
 
-  if (@available(iOS 11.0, *)) {
-    _safariAuthenticationVC = nil;
-  } else if (@available(iOS 12.0, *)) {
+  if (@available(iOS 12.0, *)) {
     _webAuthenticationVC = nil;
-  } else if (@available(iOS 9.0, *)) {
-    _safariVC = nil;
   }
+  // TODO: Remove this when the deployment target becomes iOS 12+
+  // `_safariAuthenticationVC` is cleared unconditionally just in case;
+  // It’s supposed to be assigned only when run on iOS 11.
+  _safariAuthenticationVC = nil;
 }
 
-- (void)safariViewControllerDidFinish:(SFSafariViewController *)controller NS_AVAILABLE_IOS(9.0) {
+- (void)safariViewControllerDidFinish:(SFSafariViewController *)controller {
```

Comment:
Not sure if we still need this. The `SFSafariViewControllerDelegate` conformance is part of the interface, but removing it doesn’t seem to result in any warnings. I kept the conformance and the method just in case.

Reply:
SGTM